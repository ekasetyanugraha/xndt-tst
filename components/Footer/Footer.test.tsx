import React from 'react';
import { render, screen } from '@testing-library/react';
import Footer from '.';

describe('Component: Footer', () => {
  it('should match snapshot', () => {
    render(<Footer />);

    expect(screen.getByTestId('footer')).toMatchSnapshot();
  });
});
