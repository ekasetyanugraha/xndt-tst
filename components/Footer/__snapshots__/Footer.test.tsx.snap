// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Footer should match snapshot 1`] = `
<footer
  class="footer"
  data-testid="footer"
/>
`;
